### PR TITLE
Set gmysql-dnssec parameter (DNSSEC support)

### PR DESCRIPTION
**Pull request by rfrenayworldstream**
Enable DNSSEC support (if desired). Default is no

### CODE REVIEW

**Comment by psi-4ward** [comment]
Pls meantion this in the readme

**Comment by rfrenayworldstream** [comment]
> Pls meantion this in the readme

Done :)

File: README.md [hunk 1]
```diff
@@ -48,7 +49,9 @@ $ docker run --name pdns \
   * `MYSQL_USER=root`
   * `MYSQL_PASS=root`
   * `MYSQL_DB=pdns`
+  * `MYSQL_DNSSEC=yes`
```
Comment (psi-4ward) [inline README.md]: Lets use `no` here cause the values indicates the defaults.

File: README.md [hunk 2]
```diff
@@ -48,7 +49,9 @@ $ docker run --name pdns \
...
 * Want to disable mysql initialization? Use `MYSQL_AUTOCONF=false`
+* DNSSEC is disabled by default
```
Comment (psi-4ward) [inline README.md]: => DNSSEC is disabled by default, to enable _<describe whats necessary to use it>_

**Comment by psi-4ward** [comment]
Thx, nearly ready to merge. Pls see my comments ;)

File: README.md (hunk 1 above)
Comment (rfrenayworldstream) [reply README.md]: Ok, clear

File: README.md (hunk 2 above)
Comment (rfrenayworldstream) [reply README.md]: Done

**Comment by rfrenayworldstream** [comment]
Super! Thanks for your quick merge!

**Pull request merged**